### DISCUSSION TITLE
add sysparm_display_value = false

### DISCUSSION THREAD

**Discussion by wiebew** [rbw/snow#112] [Q&A]
I would like to add "sysparm_display_value = false" to the api request how do weave this into?:
`            async for _, response in inc.stream(raw_query, offset=offset, limit=page_size, page_size=page_size, return_only=["number","opened_at","closed_at","short_description","caller_id","priority","contact_type","category","assignment_group"]):
`

**Comment by Troubladore** [comment]
You would add this to the raw_query variable. For example to qualify the Incident table by Number, I might use:

`async with Incident(client, table_name="incident") as inc:`
`     raw_query = select(Incident.number.greater_or_equals('INC0009003')).order_asc(inc.number)`
`     async for _, record in inc.stream(raw_query, limit=500, page_size=50):`
`     ...`
